### PR TITLE
RN-BZ-1894525 release notes known issue

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Added https://bugzilla.redhat.com/show_bug.cgi?id=1894525 as a known issue to the 2.5 release notes.

### CODE REVIEW

File: virt/virt-2-5-release-notes.adoc [hunk 1]
```diff
@@ -152,6 +152,8 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 //This defect is on Verified status but the fix is not ideal. This known issue is likely to be removed for 2.6.
 * If you xref:../virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc#virt-creating-vddk-image_virt-importing-vmware-vm[add a VMware Virtual Disk Development Kit (VDDK) image] to the `openshift-cnv/v2v-vmware` config map by using the web console, a *Managed resource* error message displays. You can safely ignore this error. Save the config map by clicking *Save*. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884538[*BZ#1884538*])
 
+* When nodes are placed in maintenance mode during an {product-title} cluster upgrade, virtual machines are migrated twice on node eviction, instead of just once. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888790[*BZ#1888790*])
```
Comment (stu-gott) [inline virt/virt-2-5-release-notes.adoc:155]: Suggest:

When nodes are **evicted, for instance when they** are placed in maintenance mode during an {product-title} cluster upgrade, ....

---

Rationale for the tweak is that ANY eviction will result in this situation. This includes manually evicting a node using "oc adm drain <nodename>". It just happens to be that OpenShift upgrades are the most noteworthy event that admins might not consider.

**Review by stu-gott — COMMENTED** [review]
Minor but important tweak. If this distinction gets us too far into the weeds, feel free to consider this merely a suggestion

**Review by stu-gott — APPROVED** [review]
LGTM

**Comment by ILpinto** [comment]
lgtm

**Pull request merged**